Packet++/src/NullLoopbackLayer.cpp

```diff
@@ -25,26 +26,17 @@ NullLoopbackLayer::NullLoopbackLayer(uint32_t family)
 uint32_t NullLoopbackLayer::getFamily()
 {
 	uint32_t family = *(uint32_t*)m_Data;
-    if ((family & 0xFFFF0000) != 0)
-    {
+	if ((family & 0xFFFF0000) != 0)
+	{
 		if ((family & 0xFF000000) == 0 && (family & 0x00FF0000) < 0x00060000)
-		{
-		  family >>= 16;
-		}
+			family >>= 16;
 		else
-		{
-		  family = BSWAP32(family);
-		}
-    }
-    else
-    {
-		if ((family & 0x000000FF) == 0 && (family & 0x0000FF00) < 0x00000600)
-		{
-			family = BSWAP16(family & 0xFFFF);
-		}
-    }
+			family = BSWAP32(family);
+	}
+	else if ((family & 0x000000FF) == 0 && (family & 0x0000FF00) < 0x00000600)
+		family = BSWAP16(family & 0xFFFF);

```

Comment:
why removing the parentheses? It's more readable that way...

Reply:
The existence of braces increase the number of rows in source code. This results in the need to scroll the screen. I automatically skip the braces for such cases. I recognize that the code formatting depends on personal preferences. From my point of view the second version looks more lightweight. But I can return back the previous version (with braces).
```
uint32_t NullLoopbackLayer::getFamily()
{
  uint32_t family = *(uint32_t*)m_Data;
  if ((family & 0xFFFF0000) != 0)
  {
    if ((family & 0xFF000000) == 0 && (family & 0x00FF0000) < 0x00060000)
    {
      family >>= 16;
    }
    else
    {
      family = BSWAP32(family);
    }
  }
  else if ((family & 0x000000FF) == 0 && (family & 0x0000FF00) < 0x00000600)
  {
    family = BSWAP16(family & 0xFFFF);
  }

  return family;
}
```
```
uint32_t NullLoopbackLayer::getFamily()
{
  uint32_t family = *(uint32_t*)m_Data;
  if ((family & 0xFFFF0000) != 0)
  {
    if ((family & 0xFF000000) == 0 && (family & 0x00FF0000) < 0x00060000)
      family >>= 16;
    else
      family = BSWAP32(family);
  }
  else if ((family & 0x000000FF) == 0 && (family & 0x0000FF00) < 0x00000600)
    family = BSWAP16(family & 0xFFFF);

  return family;
}
```

Reply:
The braces have been returned back

Reply:
Thanks!